package/loaders/coffee.js

```diff
@@ -1,4 +1,6 @@
+const { VERSION } = require('coffeescript')
+
 module.exports = {
   test: /\.coffee(\.erb)?$/,
-  loader: 'coffee-loader'
+  use: parseInt(VERSION, 10) >= 2 ? ['babel-loader', 'coffee-loader'] : ['coffee-loader']
```

Comment:
If you include `babel-loader` here, you loose the options passed to `babel-loader` here: https://github.com/rails/webpacker/blob/master/package/loaders/babel.js (or added in `environment.js`).

What about either patching the `babel` loader entry to add support for `.coffee` or (probably safer) copy the options passed to the babel loader?

Reply:
Oh yes, lets just copy since there isn't many.

Reply:
If you copy, it means you will need to remember to change it here if it changes in the `babel.js` file or if the user wants to change `babel-loader` options, not sure it is the best way.

Reply:
Yeah, this doesn't actually quite work, since it will fail if `coffeescript` package is not found. So, I guess for now we officially support CS1 and since the deps are now released to user space, folks who want to use CS2 can update the loader. 

```js
environment.loaders.set('coffee', //* ..... CS2 loader */)
```